Makefile

```diff
@@ -194,6 +194,7 @@ docker-generate-crds:
 		--env https_proxy=$(https_proxy) --env http_proxy=$(http_proxy) \
 		$(BUILD_IMAGE):$(OPERATOR_SDK_VERSION) /bin/bash -c 'operator-sdk generate crds'
 	@sed -i '/\- protocol/d' deploy/crds/db.orange.com_cassandraclusters_crd.yaml
+	cp deploy/crds/* */helm/*/crds/
```

Comment:
here they are copied to multi-casskop but no longer to helm/ (at the root level) 👎

Reply:
Fixed by duplicating the `cp ` to hit both `*/hem` et `./helm/` dir